package.json

```diff
@@ -9,16 +9,21 @@
     "url": "git://github.com/mozilla/moz-games.git"
   },
   "scripts": {
-    "start": "npm run prod",
+    "start": "npm run dev",
     "build": "shx rm -rf _build && shx mkdir -p _build && shx cp -R public_html/* _build",
     "preprod": "npm run build",
-    "prod": "node ./server"
+    "prod": "cross-env NODE_ENV='production' node ./server.js",
+    "dev": "cross-env NODE_ENV='development' live-server public_html --no-browser"
```

Comment:
happy to see this, I was wondering where `npm run dev` went when I was building the project first time from the readme

Reply:
np, thank you for taking a look!